plugin/src/main/java/org/jenkinsci/plugins/structs/SymbolLookup.java

```diff
@@ -152,5 +153,29 @@ public static SymbolLookup get() {
         return j.getInjector().getInstance(SymbolLookup.class);
     }
 
+    /**
+     * Get the {@link Symbol} value for the class of the given object, if the annotation is present
+     *
+     * @param o An object
```

Comment:
Or get rid of the `Object` overload and add two overloads, one taking `Descriptor` (behavior as here), one taking `Describable` (to call `.getDescriptor()` first).